### ISSUE TITLE
Blocked from starting a new domain request when there are other started domain requests

### ISSUE THREAD

**Issue by AnnaGingle** [cisagov/manage.get.gov#2927]
### Current Behavior

When I click start a new domain request I am brought to a request I've already started

### Expected Behavior

When I click start a new domain request I am brought to a brand new form with nothing filled in 

### Steps to Reproduce

1. login to a non-org account
2. Start a domain request, fill in the first 3 pages
3. Go back and start another new request

### Environment

Firefox

### Additional Context

https://github.com/user-attachments/assets/059404e4-c83b-40cc-ab90-c47572f71a03

**Comment by katypies** [comment]
@Katherine-Osos or @h-m-f-t can you confirm if this is expected? 

@PaulKuykendall thinks this is by design - you shouldn't be able to start a different request if you have one that you've already started but not submitted.

**Comment by h-m-f-t** [comment]
IIRC, there is an architectural constraint here (different than our decision to limit a person with no approved domains to submit additional requests). 

I could get behind this being a deliberate choice? Maybe we just need clearer signage.

**Comment by Katherine-Osos** [comment]
@katypies I think Paul may be remembering a different rule in the UI that's related but not quite what Anna's describing. 

Users who don't have at least one approved domain request ... but have one request pending ... cannot submit another request until the pending one has been approved (or withdrawn or rejected). However, in that scenario, the user _should_ be able to start a new request and save it as a draft. They'll see a banner that informs them the request cannot be submitted yet, as illustrated in [this Figma](https://www.figma.com/design/xFtGLHVrhp0lvwh0gYWnbx/.gov-registar?node-id=2211-25394&node-type=canvas&t=hAJZ96JJnrSia0uq-0).

Once a user has an approved domain, they can request as many domains as they want with no blockers.

**Comment by Katherine-Osos** [comment]
When someone clicks the  button to "Start a new request," it should be blank. I kinda feel like this was a bug we ran into previously and fixed. Calling in @abroddrick to see if she remembers that.

**Comment by abroddrick** [comment]
Yes run into this before. It was a session issue, and guess what we changed fairly recently... session storage. We may have resurrected the old bug
